### PR TITLE
Add cache for annotation, serializer and validation

### PR DESCRIPTION
**Pull request by Nyholm**
Should the SymfonyValidatorBridge be in a separate package? 
Pros: 
- It can be resused
- It is similar to the DoctrineCacheBridge

Cons: 
- Nobody wants to reuse it
- It is dead simple to write

### CODE REVIEW

File: src/DependencyInjection/CacheExtension.php [hunk 1]
```diff
@@ -36,12 +38,35 @@ public function load(array $configs, ContainerBuilder $container)
...
+        foreach (['router', 'session', 'doctrine', 'logging', 'annotation', 'serializer', 'validation'] as $section) {
             if ($config[$section]['enabled']) {
                 $container->setParameter('cache.'.$section, $config[$section]);
             }
         }
 
+        if ($config['annotation']['enabled']) {
+            $this->verifyDoctrineBridgeExists('annotation');
+            $container->register('cache.service.annotation', DoctrineCacheBridge::class)
+                ->setFactory('Cache\CacheBundle\Factory\AnnotationFactory::get')
```
Comment (cryptiklemur) [inline src/DependencyInjection/CacheExtension.php]: these should be arrays, `[AnnotationFactory::class, 'get']`

**Comment by cryptiklemur** [comment]
I think they should stay in this package

**Comment by Nyholm** [comment]
Agreed. Im about to create another bridge for the session

File: src/DependencyInjection/CacheExtension.php (hunk 1 above)
Comment (Nyholm) [inline src/DependencyInjection/CacheExtension.php]: It does not have to. It works as it is. 
I did not know you could do like you suggested, Your way is better. I'll change

**Comment by cryptiklemur** [comment]
Go ahead and merge when ready, looks good

**Pull request merged**